app/src/generator/Radii/Radii.tsx

```diff
@@ -16,19 +16,57 @@ const Radii = () => {
   const [currValues, setCurrValues] = useState<Map<string, string | number>>(
     new Map<string, string | number>()
   );
+  const currValuesRef = useRef<Map<string, string | number>>(currValues);
 
-  const valueChangedHandler = (spacing: string, value: string) => {
+  const valueChangedHandler = (radii: string, value: string) => {
```

Comment:
same as above
```suggestion
  const valueChangedHandler = (radius: Radius, value: string) => {
```
This way we can avoid the evil `as` casts

Reply:
Cool, I'll make those changes